src/main/java/pl/cleankod/exchange/core/domain/Account.java

```diff
@@ -37,4 +38,8 @@ public static Number of(String value) {
             return new Number(value);
         }
     }
+
+    public AccountDto toDto() {
```

Comment:
We should keep the domain as pure as possible. Other infra layers should depend on domain, not the other way around.